crux-mir/README.md

```diff
@@ -11,27 +11,30 @@ functionality.
 
 ## Preliminaries
 
-You must have the most recent version of the `mir-json` executable in your
-path.  See [the `mir-json` README][mir-json-readme] for installation
-instructions.
-
 `crux-mir` uses several submodules, so make sure they're initialized:
 
     $ git submodule update --init
 
+Next, navigate to the `crucible/dependencies/mir-json` directory and install
+`mir-json` according to the instructions in [the `mir-json`
+README][mir-json-readme].
+
 [mir-json-readme]: https://github.com/GaloisInc/mir-json#readme
 
 
 ## Installation
 
-Use ghc-8.4.4 or ghc-8.6.5.
+Use ghc-8.4.4 or ghc-8.6.5.  From the `crux-mir` directory, run:
```

Comment:
Oh, I think 8.8.4 will also work. It's in the CI now.

Reply:
I added 8.8.4, and also removed 8.4.4, which isn't covered by CI any more